src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const rentCost = 40;
+  const smallDiscount = 20;
+  const bigDiscount = 50;
```

Comment:
i think better to use this names for this variables

`const threeDaysDiscount
const sevenDaysDiscount`

Reply:
maybe it will be better